server/queries/variants.js

```diff
@@ -0,0 +1,20 @@
+ const db = require("../utils/dbConfig");
+
+// function insertVariants(params) {
+//   return db("variants").insert({
+//     name: params.name,
+//     description: params.description,
+//     category_id: params.category,
+//   });
+// }
+function updateVariants(params) {
```

Comment:
can I update multiple variants using this function? if no, then you will have correct the spelling of this function.